pkg/api/types.go

```diff
@@ -504,20 +505,31 @@ func (l *VolumeList) Set(value string) error {
 	if len(value) == 0 {
 		return errors.New("invalid format, must be source:destination")
 	}
+
+	var pos int
+	isWindows, _ := regexp.MatchString("^[A-Za-z]:\\.*", value)
+	if isWindows {
+		pos = strings.IndexAny(value[3:], ":") + 3
```

Comment:
Very suspicious to use `strings.IndexAny` when [`strings.Index`](https://golang.org/pkg/strings/#Index) would do it: we're looking for a single character in the input.

The "magic number" 3 is better given a name, what does it mean?
Seems unclear for people reading the code why on Windows there is this offset.

Also suspicious that we're changing from `strings.LastIndex` to `strings.IndexAny` / `strings.Index`, as they behave quite differently when there are more occurrences of ":" in the input.